### ISSUE TITLE
post-merge hexy: Several export-* primitives throw exceptions

### ISSUE THREAD

**Issue by qiemem** [NetLogo/NetLogo#967]
Tested on Xubuntu 15.10. This could definitely be an OS specific thing. To reproduce, run any of the following primitives:

- `export-interface`
- `export-plot`
- `export-all-plots`

They all produce the following exception:

```
java.security.PrivilegedActionException: java.security.PrivilegedActionException: java.lang.ClassNotFoundException: org.nlogo.prim.etc._exportplot
 at java.security.AccessController.doPrivileged(Native Method)
 at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
 at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
 at org.GNOME.Accessibility.AtkWrapper$5.dispatchEvent(AtkWrapper.java:697)
 at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
 at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
 at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
 at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
 at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
 at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.security.PrivilegedActionException: java.lang.ClassNotFoundException: org.nlogo.prim.etc._exportplot
 at java.security.AccessController.doPrivileged(Native Method)
 at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
 at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
 at java.awt.EventQueue$4.run(EventQueue.java:731)
 at java.awt.EventQueue$4.run(EventQueue.java:729)
 ... 10 more
Caused by: java.lang.ClassNotFoundException: org.nlogo.prim.etc._exportplot
 at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
 at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
 at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
 at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
 at java.lang.Class.forName0(Native Method)
 at java.lang.Class.forName(Class.java:264)
 at org.nlogo.compiler.Backifier.fallback(Backifier.scala:119)
 at org.nlogo.compiler.Backifier.apply(Backifier.scala:144)
 at org.nlogo.compiler.ASTBackifier.backify(ASTBackifier.scala:35)
 at org.nlogo.compiler.ASTBackifier$$anonfun$backify$1.apply(ASTBackifier.scala:29)
 at org.nlogo.compiler.ASTBackifier$$anonfun$backify$1.apply(ASTBackifier.scala:29)
 at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
 at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
 at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
 at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
 at scala.collection.TraversableLike$class.map(TraversableLike.scala:245)
 at scala.collection.AbstractTraversable.map(Traversable.scala:104)
 at org.nlogo.compiler.ASTBackifier.backify(ASTBackifier.scala:29)
 at org.nlogo.compiler.ASTBackifier.backifyProcedure(ASTBackifier.scala:18)
 at org.nlogo.compiler.CompilerBridge$$anonfun$3.apply(CompilerBridge.scala:28)
 at org.nlogo.compiler.CompilerBridge$$anonfun$3.apply(CompilerBridge.scala:28)
 at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:48)
 at scala.Function2$$anonfun$tupled$1.apply(Function2.scala:47)
 at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
 at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:245)
 at scala.collection.immutable.List.foreach(List.scala:381)
 at scala.collection.TraversableLike$class.map(TraversableLike.scala:245)
 at scala.collection.immutable.List.map(List.scala:285)
 at org.nlogo.compiler.CompilerBridge$.apply(CompilerBridge.scala:28)
 at org.nlogo.compiler.CompilerMain$.compile(CompilerMain.scala:30)
 at org.nlogo.compiler.Compiler.compileMoreCode(Compiler.scala:64)
 at org.nlogo.window.CompilerManager.handle(CompilerManager.java:235)
 at org.nlogo.window.Events$CompileMoreSourceEvent.beHandledBy(Events.java:190)
 at org.nlogo.window.Event.doRaise(Event.java:198)
 at org.nlogo.window.Event.raise(Event.java:122)
 at org.nlogo.window.JobWidget.source(JobWidget.scala:120)
 at org.nlogo.app.CommandLine.executeCurrentBuffer(CommandLine.scala:157)
 at org.nlogo.app.CommandLine.keyPressed(CommandLine.scala:118)
 at java.awt.Component.processKeyEvent(Component.java:6493)
 at javax.swing.JComponent.processKeyEvent(JComponent.java:2832)
 at java.awt.Component.processEvent(Component.java:6312)
 at java.awt.Container.processEvent(Container.java:2236)
 at java.awt.Component.dispatchEventImpl(Component.java:4891)
 at java.awt.Container.dispatchEventImpl(Container.java:2294)
 at java.awt.Component.dispatchEvent(Component.java:4713)
 at java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1954)
 at java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:806)
 at java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1074)
 at java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:945)
 at java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:771)
 at java.awt.Component.dispatchEventImpl(Component.java:4762)
 at java.awt.Container.dispatchEventImpl(Container.java:2294)
 at java.awt.Window.dispatchEventImpl(Window.java:2750)
 at java.awt.Component.dispatchEvent(Component.java:4713)
 at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
 at java.awt.EventQueue.access$500(EventQueue.java:97)
 at java.awt.EventQueue$3.run(EventQueue.java:709)
 at java.awt.EventQueue$3.run(EventQueue.java:703)
 ... 15 more
```

I confirmed that they worked correctly in the constructionism preview.

Note that `export-view` works just fine.

**Comment by qiemem** [comment]
To clarify, they all produce exceptions *like* that one. Obviously, the class referred to at the top is specific to the primitive.

**Comment by mrerrormessage** [comment]
This should be a simple fix - all three just need special case translations [in this map in Backifier](https://github.com/NetLogo/NetLogo/blob/hexy/netlogo-gui/src/main/compiler/Backifier.scala#L13). Ideally, this wouldn't be needed, but enough primitives are in different modules that it's the easiest solution for the moment.

**Comment by mrerrormessage** [comment]
Fixed in #973
